A2-service-configs-in-dns.md

```diff
@@ -48,24 +48,38 @@ determine which choice will be selected by a given client:
     // Criteria used to select this choice.
     // If a field is absent or empty, it matches all clients.
     // All fields must match a client for this choice to be selected.
+    // If any unexpected field name is present in this object, the entire
+    // choice is considered invalid.
     //
     // Client language(s): a list of strings (e.g., 'c++', 'java', 'go',
-    // 'python', etc).
+    // 'python', etc).  Each string is case insensitive.
     'clientLanguage': [string],
     // Percentage: integer from 0 to 100 indicating the percentage of
-    // clients that should use this choice.
+    // clients that should use this choice.  If present, the number must
+    // match the regular expression ^0|[0-9]|[1-9][0-9]|100$
```

Comment:
The "`0|`" should not be necessary given the "`[0-9]`".

This also works IIUC:
`^[1-9]?[0-9]|100$`